tinyagentos/installers/download_installer.py

```
@@ -28,16 +70,54 @@ async def download_file(
+
     dest.parent.mkdir(parents=True, exist_ok=True)
     import time as _time
+
+    sha = hashlib.sha256()
+    downloaded = 0
+    total = 0
     last_cb = 0.0
-    async with httpx.AsyncClient(timeout=None, follow_redirects=True) as client:
-        async with client.stream("GET", url) as resp:
+
+    async with httpx.AsyncClient(timeout=None, follow_redirects=False) as client:
```

Ruff (0.15.21)

[error] Probable use of `httpx` call with timeout set to `None`

(S113)